ansible/vars.yml

```diff
@@ -100,13 +101,15 @@ libsodium_release_checksum: sha1:795b73e3f92a362fabee238a71735579bf46bb97
 pgsodium_release: "3.0.4"
 pgsodium_release_checksum: sha1:f08e9ac109ab5e6fc8b15ea21b26f88f451a2070
 
-pg_graphql_pgx_version: "0.5.2"
-pg_graphql_release: "v0.5.0"
+#TODO: update versions when both stable pgx and pg_graphql versions are released
+pg_graphql_pgx_version: "0.6.0-alpha.1"
+pg_graphql_release: "pg15_support"
 
-pg_jsonschema_pgx_version: "0.5.6"
-pg_jsonschema_release: "v0.1.1"
+#TODO: update versions when both stable pgx and pg_jsonschema versions are released
+pg_jsonschema_pgx_version: "0.6.0-alpha.1"
+pg_jsonschema_release: "pg15_support"
```

Comment:
Needs update to point to proper pg_graphql and pg_jsonschema releases once PG15 support is merged